src/api/test/functional/product_test.rb

```diff
@@ -10,6 +10,49 @@ def setup
     reset_auth
   end
 
+  def _simple_product_file_calls(prefix)
```

Comment:
On Tuesday 19 January 2016, 09:44:36 wrote Henne Vogelsang:
> > @@ -10,6 +10,49 @@ def setup
> >      reset_auth
> >    end
> >  
> > +  def _simple_product_file_calls(prefix)
> 
> What are you trying to say with the underscore as a prefix for the method name?

it is (should be) a private method 

-- 

Adrian Schroeter
email: adrian@suse.de

SUSE Linux GmbH, GF: Felix Imend�rffer, Jane Smithard, Graham Norton, HRB 21284 (AG N�rnberg) 
 
Maxfeldstra�e 5                         
90409 N�rnberg 
Germany